### DISCUSSION TITLE
Is it encouraged to modify shadCN components?

### DISCUSSION THREAD

**Discussion by dennisvaga** [shadcn-ui/ui#5644] [Q&A]
Hey everyone,

I have a card.tsx component that I want to modify. I’m wondering if it's advisable to modify the component directly, or if there’s a better way to approach this while following best practices.

Any advice would be appreciated!

**Comment by shadcn** [comment]
Yes. It's a starting point. You install the code to your codebase and update it to fit your needs. You're encouraged to update it.

**Reply by dennisvaga** [reply]
Thanks! That clears it up for me. Appreciate the confirmation!
